src/widgets/radio.jsx

```diff
@@ -598,7 +690,7 @@ var RadioEditor = React.createClass({
     },
 
     focus: function() {
-        this.refs.editor0.focus();
+        this.refs.editor0.refs['content-editor'].focus();
```

Comment:
I think this should be

    this.refs['choice-editor0'].refs['content-editor'].focus()

now that we've changed the editor keys from "editor0"..."editor3" to "choice-editor0"..."choice-editor3"